recipes/react-router/app/puck.config.tsx

```diff
@@ -0,0 +1,23 @@
+import type { Config } from "@measured/puck";
+
+type Props = {
+  HeadingBlock: { title: string };
+};
+
+export const config: Config<Props> = {
+  components: {
+    HeadingBlock: {
+      fields: {
+        title: { type: "text" },
+      },
+      defaultProps: {
+        title: "Heading",
+      },
+      render: ({ title }) => (
+        <div style={{ padding: 64 }}>
+          <h1>{title}</h1>
+        </div>
+      ),
+    },
+  },
+};
```

Comment:
Can this file move up a directory? I know that the TS compiler might require the files to be in certain directories to run properly, so wasn't sure if this was deliberate or not.